cmd/update-main.go

```diff
@@ -475,12 +475,12 @@ func doUpdate(sha256Hex string, latestReleaseTime time.Time, ok bool) (successMs
 	defer resp.Body.Close()
 
 	// FIXME: add support for gpg verification as well.
-	if err = update.RollbackError(update.Apply(resp.Body,
+	if err = update.Apply(resp.Body,
 		update.Options{
 			Hash:     crypto.SHA256,
 			Checksum: sha256Sum,
 		},
-	)); err != nil {
+	); err != nil {
```

Comment:
This is really confusing: we return success  message, when we detect error condition ????

Reply:
@ebozduman Although its named as successMsg, it is giving information about update status. I will change the name of the variable.